src/arsenic/connection.py

```diff
@@ -72,6 +67,19 @@ def strip_auth(url: str) -> str:
     )
 
 
+def check_response_error(*, status: int, data: Any) -> None:
+    if status >= 400:
+        errors.raise_exception(data, status)
+    if not isinstance(data, dict):
+        return
+    data_status = data.get("status", None)
+    if data_status is None:
+        return
+    if data_status == constants.STATUS_SUCCESS:
+        return
+    errors.raise_exception(data, status)
```

Comment:
The blob mentioned in the issue seems to have "value" but not "status".
What am I missing?

Reply:
This is unrelated to #100, but I've noticed that when using a remote driver (eg browserstack) and something goes wrong on their end, the *next* operation will fail (with "Session terminated or not started") which is confusing. The actual error is one operation before that but hidden in the data and the status code is 200 for some reason.